env0/resource_project_policy.go

```diff
@@ -185,7 +189,9 @@ func resourcePolicyReset(ctx context.Context, d *schema.ResourceData, meta inter
 	}
 
 	d.SetId(projectId)
-	setPolicySchema(d, policy)
+	if err := writeResourceData(&policy, d); err != nil {
```

Comment:
bug - writeResourceData was failing and was not caught. Adding "tfschema:"-"" fixes it.